man/complete.tbl_lazy.Rd

```diff
@@ -4,7 +4,7 @@
 \alias{complete.tbl_lazy}
 \title{Complete a SQL table with missing combinations of data}
 \usage{
-complete.tbl_lazy(data, ..., fill = list())
```

Comment:
I think something idiosyncratic about your setup made this change. To make it work persistently (and for me) you need to use a hack:

```
Roxygen: {library(tidyr); list(markdown = TRUE)}
```